src/main/java/org/pac4j/dropwizard/Pac4jBundle.java

```diff
@@ -43,8 +44,39 @@ public final void run(T configuration, Environment environment)
             }
         }
 
-        environment.jersey().register(new Pac4JSecurityFeature(config));
+        String urlPrefix = getUrlPrefix(configuration, environment);
+
+        environment.jersey()
+                .register(new Pac4JSecurityFeature(config, urlPrefix));
         environment.jersey()
                 .register(new Pac4JValueFactoryProvider.Binder(config));
     }
+
+    private String getUrlPrefix(T configuration, Environment environment) {
+        // the context path on the
+        String cPath = environment.getApplicationContext().getContextPath();
+        String sPath = ((AbstractServerFactory) configuration
+                .getServerFactory()).getJerseyRootPath().orElse("");
+
+        return (clean(cPath) + clean(sPath)).replace("//", "/");
+    }
+
+    private String clean(String path) {
```

Comment:
This is a lot of logics to clean the path, but I guess there is no better option.